README.md

```diff
@@ -1,2 +1,116 @@
-# community
-CoreDash Governance
+# Welcome to the CoreDash Community!
+
+With the recent license changes of the Grafana project, there are quite a few
+community end users as well as monitoring vendors who are interested in a
+PromQL visualization and dashboarding solution that is under the Apache-2.0 
+license, owned by the Linux Foundation, and governed by a mix of community
+end users and vendors. The CoreDash Project is designed as a place where we can
+work together on this solution. Please note that CoreDash is not designed to be
+a fork of Grafana, but will instead take a more componetized approach as a
+collection of subprojects that users can select from to meet their needs.
+
+CoreDash is the current working name / code name for this effort, but we expect
+to rename it after we've made a bit more progress and better defined what we
```

Comment:
Probably not ? From the governance file: 

> The CoreDash umbrella project is composed as a federation of individual projects,
some independent and some interdependent, each of which focuses on some aspect
of CoreDash. Our governance reflects this federated structure.

For me it's clear that CoreDash is here for legal purpose and to drive any initiative around this topic. It hasn't the goal to be the final of the product(s). 

But probably I missed something

Reply:
I think this is still true. I think about CoreDash as a collection of projects or an umbrella group that provides a legal entity and structure for the overall effort. I don't necessarily think it will become a brand, but this governance structure allows us to provide an overall entity (CoreDash) where we can work on individual projects (aka subprojects.) By having this structure, we have a steering committee to make decisions about which projects to accept so that we ensure that we don't become a dumping ground for things that aren't going to be useful for people.